ocaml/xapi/context.ml

```diff
@@ -148,6 +148,13 @@ let __destroy_task : (__context:t -> API.ref_task -> unit) ref =
 
 let string_of_task __context = __context.dbg
 
+let string_of_task_and_tracing __context =
+  match Tracing.t_to_string_opt __context.tracing with
+  | None ->
+      __context.dbg
+  | Some tracing ->
+      __context.dbg ^ "\x00" ^ tracing
```

Comment:
Is there a plan to replace `string_of_task` with this function in all situations? I'd try to use the final name instead so the namechange happens only once. I think `to_string` would a good candidate

Reply:
This came up last Tuesday, There is no clear iea yet on the best way to approach getting tracing into xenopsd, changing the type or adding parameter would require a lot of changes. It terms of naming, I see your point this name is convoluted.

Reply:
> I think to_string would a good candidate

It's not a string of the whole context though. Just of the task and tracing info in the context.